core/debug/inc/debug.h

```diff
@@ -18,8 +18,19 @@
 #define __DEBUG_H__
 
 #include <uvisor.h>
+#include "debug_exports.h"
 #include "halt.h"
 
+/* Debug box handle.
+ * This is used internally by uVisor to keep track of the registered debug box
+ * and to access its driver when needed. */
+typedef struct TDebugBox {
+    const TUvisorDebugDriver *driver;
+    uint8_t box_id;
```

Comment:
Externally, we use int for box_id (since the Box ID API introduction). Internally, we currently use uint8_t.